export/examples/http/main.go

```diff
@@ -0,0 +1,72 @@
+package main
```

Comment:
what do you think about moving `examples` to be a top-level directory rather than having it namespaced under `export` ?

Reply:
so the reason I put it here was because I imagined a top level `examples` directory using an example of both collectors and exporters, but if we're going to ignore collectors for now then I think it would be a good idea to move examples to top-level